tutorials/synthetic-data-hello-world/Synthetic Data Generation - Hello World Examples.ipynb

```diff
@@ -0,0 +1,1396 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 101,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "messages = [\n",
+    "    {\n",
+    "        \"role\": \"user\", \n",
+    "        \"content\": \"I am going to Paris, what should I see?\"\n",
+    "    },\n",
+    "    {\n",
+    "        \"role\": \"assistant\",\n",
+    "        \"content\": \"Ah, Paris, the City of Light! There are so many amazing things to see and do in this beautiful city...\",\n",
+    "    },\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After that, we can simply fire off our request!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 103,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'helpfulness': 1.4765625, 'correctness': 1.6171875, 'coherence': 3.21875, 'complexity': 0.640625, 'verbosity': 0.365234375}\n"
+     ]
+    }
+   ],
+   "source": [
+    "rewards = curator_openai_client.query_reward_model(messages=messages, model=model)\n",
+    "print(rewards)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Nemotron-4 340B Reward model will provide us the scores (between 0 and 4) for each of the 5 SteerLM attributes:\n",
+    "\n",
+    "- **Helpfulness:** Overall helpfulness of the response to the prompt.\n",
+    "- **Correctness:** Inclusion of all pertinent facts without errors.\n",
+    "- **Coherence:** Consistency and clarity of expression.\n",
+    "- **Complexity:** Intellectual depth required to write response (i.e. whether the response can be written by anyone with basic language competency or requires deep domain expertise).\n",
+    "- **Verbosity:** Amount of detail included in the response, relative to what is asked for in the prompt.\n",
+    "\n",
+    "These can be used as a filter for any of the individual attributes, or utilized to verify specific attributes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using The `NemotronGenerator`\n",
+    "\n",
+    "The NeMo Curator Synthetic Data Generation (SDG) features are primarily accessed through the `NemotronGenerator` class. \n",
+    "\n",
+    "This useful wrapped helps expose both: \n",
+    "\n",
+    "1. Pre-built SDG pipelines \n",
+    "2. A number of specific generation utilities, which we'll explore in the following section of the notebook.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 117,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nemo_curator.synthetic import NemotronGenerator\n",
+    "\n",
+    "generator = NemotronGenerator(curator_openai_client)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you'd like to skip forward to a specific pipeline, you can find them here:\n",
```

Comment:
Do these links work locally? On Github they do not

Reply:
They do work locally, yes.